lib/components/app/desktop-nav.tsx

```diff
@@ -1,14 +1,14 @@
-import React from 'react'
-import { Nav, Navbar } from 'react-bootstrap'
 import { connect } from 'react-redux'
+import { Nav, Navbar } from 'react-bootstrap'
+import React from 'react'
 
-import NavLoginButtonAuth0 from '../user/nav-login-button-auth0.js'
 import { accountLinks, getAuth0Config } from '../../util/auth'
 import { DEFAULT_APP_TITLE } from '../../util/constants'
+import NavLoginButtonAuth0 from '../user/nav-login-button-auth0'
 
 import AppMenu from './app-menu'
+import LocaleSelector from './locale-selector'
```

Comment:
Is the locale selector supposed to be available in the mobile view too? (Right now it is not.)

Reply:
Updated to be included in the mobile view in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/2ba2f5eeb9da864347789e418be2bc16ab16ce6b